<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bestik</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/recources/window_icon.png</normaloff>:/recources/window_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="formula"/>
        </item>
        <item>
         <widget class="QTabWidget" name="tabGraphSettings">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="widget">
           <attribute name="title">
            <string>Таблица</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="rowCount">
               <number>40</number>
              </property>
              <property name="columnCount">
               <number>20</number>
              </property>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Приборы</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableWidget" name="instrumentsTable">
              <property name="columnCount">
               <number>3</number>
              </property>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Настройки графиков</string>
           </attribute>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>291</width>
              <height>240</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Цвет:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Толщина линий:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Стиль линий:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Стиль точек:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Размер точек:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Подпись OX:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Подпись OY:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="line_tab">
        <attribute name="title">
         <string>График</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCustomPlot" name="linePlot" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="hist_tab">
        <attribute name="title">
         <string>Гистограмма</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QCustomPlot" name="histogram" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="scat_tab">
        <attribute name="title">
         <string>Скаттерплот</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QCustomPlot" name="scatterPlot" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_ODF"/>
    <addaction name="export_CSV"/>
    <addaction name="import_CSV"/>
   </widget>
   <addaction name="Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>меню</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Файл</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_ODF">
   <property name="text">
    <string>Создать отчет</string>
   </property>
   <property name="toolTip">
    <string>Экспорт в ODF</string>
   </property>
  </action>
  <action name="export_CSV">
   <property name="text">
    <string>Экспорт CSV</string>
   </property>
  </action>
  <action name="import_CSV">
   <property name="text">
    <string>Импорт CSV</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
